<ui version="4.0" >
 <class>siteManager</class>
 <widget class="QDialog" name="siteManager" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Address Book</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <layout class="QGridLayout" >
     <item row="0" column="0" colspan="2" >
      <widget class="QListWidget" name="siteList" />
     </item>
     <item row="1" column="0" >
      <widget class="QPushButton" name="upButton" >
       <property name="text" >
        <string>&amp;Up</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QPushButton" name="downButton" >
       <property name="text" >
        <string>&amp;Down</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" >
    <widget class="QGroupBox" name="editGroup" >
     <property name="title" >
      <string/>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="nameLabel" >
        <property name="text" >
         <string>Site Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3" >
       <widget class="QLineEdit" name="nameEdit" />
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="addrLabel" >
        <property name="text" >
         <string>Address</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="3" >
       <widget class="QLineEdit" name="addrEdit" />
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="protocolLabel" >
        <property name="text" >
         <string>Protocol</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QComboBox" name="protocolCombo" >
        <item>
         <property name="text" >
          <string>Telnet</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>SSH1</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>SSH2</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QCheckBox" name="portCheck" >
        <property name="text" >
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3" >
       <widget class="QLineEdit" name="portEdit" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="hostTypeLabel" >
        <property name="text" >
         <string>Host Type</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QComboBox" name="hostTypeCombo" >
        <item>
         <property name="text" >
          <string>BBS</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>*NIX</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QCheckBox" name="proxyCheck" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Use Proxy</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="QPushButton" name="proxyButton" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Setting...</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" >
       <widget class="QCheckBox" name="autoLoginCheck" >
        <property name="text" >
         <string>Auto Login</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" >
       <widget class="QPushButton" name="autoLoginButton" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Setting...</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" >
       <widget class="QLabel" name="label_10" >
        <property name="text" >
         <string>Font Preview</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="3" >
       <widget class="QLabel" name="fontPreviewer" >
        <property name="autoFillBackground" >
         <bool>true</bool>
        </property>
        <property name="text" >
         <string>Font Previewer</string>
        </property>
        <property name="textFormat" >
         <enum>Qt::RichText</enum>
        </property>
        <property name="alignment" >
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="4" >
       <layout class="QHBoxLayout" >
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>55</width>
            <height>25</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="resetButton" >
          <property name="text" >
           <string>&amp;Reset</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>54</width>
            <height>25</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="advanceButton" >
          <property name="text" >
           <string>A&amp;dvance</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="7" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="newButton" >
       <property name="text" >
        <string>&amp;New</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton" >
       <property name="text" >
        <string>&amp;Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="connectButton" >
       <property name="text" >
        <string>C&amp;onnect</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton" >
       <property name="text" >
        <string>&amp;Save</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton" >
       <property name="text" >
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>553</y>
     <width>31</width>
     <height>0</height>
    </rect>
   </property>
   <property name="maximumSize" >
    <size>
     <width>16777215</width>
     <height>0</height>
    </size>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>120</x>
     <y>553</y>
     <width>30</width>
     <height>0</height>
    </rect>
   </property>
   <property name="maximumSize" >
    <size>
     <width>16777215</width>
     <height>0</height>
    </size>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>156</x>
     <y>553</y>
     <width>78</width>
     <height>0</height>
    </rect>
   </property>
   <property name="maximumSize" >
    <size>
     <width>16777215</width>
     <height>0</height>
    </size>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4" >
   <property name="geometry" >
    <rect>
     <x>240</x>
     <y>553</y>
     <width>69</width>
     <height>0</height>
    </rect>
   </property>
   <property name="maximumSize" >
    <size>
     <width>16777215</width>
     <height>0</height>
    </size>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5" >
   <property name="geometry" >
    <rect>
     <x>410</x>
     <y>553</y>
     <width>34</width>
     <height>0</height>
    </rect>
   </property>
   <property name="maximumSize" >
    <size>
     <width>16777215</width>
     <height>0</height>
    </size>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6" >
   <property name="geometry" >
    <rect>
     <x>369</x>
     <y>553</y>
     <width>35</width>
     <height>0</height>
    </rect>
   </property>
   <property name="maximumSize" >
    <size>
     <width>16777215</width>
     <height>0</height>
    </size>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7" >
   <property name="geometry" >
    <rect>
     <x>46</x>
     <y>553</y>
     <width>30</width>
     <height>0</height>
    </rect>
   </property>
   <property name="maximumSize" >
    <size>
     <width>16777215</width>
     <height>0</height>
    </size>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8" >
   <property name="geometry" >
    <rect>
     <x>82</x>
     <y>553</y>
     <width>32</width>
     <height>0</height>
    </rect>
   </property>
   <property name="maximumSize" >
    <size>
     <width>16777215</width>
     <height>0</height>
    </size>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9" >
   <property name="geometry" >
    <rect>
     <x>315</x>
     <y>553</y>
     <width>48</width>
     <height>0</height>
    </rect>
   </property>
   <property name="maximumSize" >
    <size>
     <width>16777215</width>
     <height>0</height>
    </size>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>siteList</tabstop>
  <tabstop>nameEdit</tabstop>
  <tabstop>addrEdit</tabstop>
  <tabstop>protocolCombo</tabstop>
  <tabstop>portCheck</tabstop>
  <tabstop>portEdit</tabstop>
  <tabstop>hostTypeCombo</tabstop>
  <tabstop>proxyCheck</tabstop>
  <tabstop>autoLoginCheck</tabstop>
  <tabstop>connectButton</tabstop>
  <tabstop>upButton</tabstop>
  <tabstop>downButton</tabstop>
  <tabstop>resetButton</tabstop>
  <tabstop>advanceButton</tabstop>
  <tabstop>newButton</tabstop>
  <tabstop>deleteButton</tabstop>
  <tabstop>applyButton</tabstop>
  <tabstop>closeButton</tabstop>
  <tabstop>proxyButton</tabstop>
  <tabstop>autoLoginButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>portCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>portEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>372</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel" >
     <x>477</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>proxyCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>proxyButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>235</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel" >
     <x>285</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autoLoginCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>autoLoginButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>232</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel" >
     <x>361</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
